doc/sampler.rst

```diff
@@ -105,23 +105,62 @@ How to setup a Redis based distributed cluster
 ----------------------------------------------
 
 
-Step 1: Start a Redis server without password authentication
+Step 1: Reconfigure the redis.conf file
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Start one some machine, which is reachable by the machine running the pyABC
-main application and by the workers, a Redis server, disabling password
-authentication:
+It is advised to run Redis server in protected mode. Authenticated
+communication will allow only for authenticated access to communicated with
+the server, and reject all unauthorised access. To run Redis server with
+authentication required you need first to modify the redis.conf file.
+
+The redis.conf is the file that contains Redis configuration. Usually,
+it can be found on `/etc/redis/`. to allow safe and secure communication,
+redis.conf file should be reconfigure as follow:
+
+1. The Redis server should be configured in a way that allows it to bind to
+network interfaces other than localhost (127.0.0.1). To enable that, be sure
+that the bind configuration option is either commented out or modified to an
+appropriate network interface IP address.
 
 
 .. code:: bash
 
-   redis-server --protected-mode no
+    #bind 127.0.0.1
 
+2. The password authentication must be enabled. To configure that, be sure
+that the masterauth and requirepass configuration options are uncommented, and
+their values are the SAME. Note: it advised to select complex password string
+(e.g., thW0K5tB4URO5a9wsykBH8ja4AdwkQcw).
```

Comment:
I wouldn't give an example here. Users might just copy it, which would make sense very unsecure again ;)